.travis.yml

```diff
@@ -114,7 +114,7 @@ install:
 
 before_script:
   - export GAPROOT="$HOME/gap"
-  - scripts/travis-build-dependencies.sh
+  - ci/travis-build-dependencies.sh
```

Comment:
Our actions really shouldn't put anything inside the package dir, to prevent clashes like this. Some temp location outside should be used instead. Or if that's infeasible, at least a really unique dir name should be used. CC @ssiccha

Reply:
Yeah I renamed this because `pkg-ci-scripts` was getting cloned into a `scripts` directory.

Reply:
Although for now I'm using a forked version of one of the actions https://github.com/wilfwilson/setup-gap-for-packages/blob/master/action.yml - I'll submit a PR to the  `gap-actions` repo soon.

Reply:
https://github.com/gap-actions/setup-gap-for-packages/pull/6